commercetools-models/src/main/java/io/sphere/sdk/orders/OrderFromCartDraft.java

```diff
@@ -43,4 +54,12 @@ static OrderFromCartDraft of(final Versioned<Cart> cart, @Nullable final String
     static OrderFromCartDraft of(final Versioned<Cart> cart) {
         return OrderFromCartDraftDsl.of(cart.getId(), cart.getVersion(), null, null);
```

Comment:
please add here something like `ResourceIdentifier.of(cart.getId())` so that in future the deprecated method is not used anymore

Reply:
just to be clear: replacing the parameter for the of method `cart.getId()` with `ResourceIdentifier.ofId(cart.getId())`, should it be enough?

Reply:
Would it be better maybe put @Deprecated here and the link to the method which replaces it?

Reply:
The method itself is not deprecated, only the structure send to the API. I adjusted it accordingly